Package.swift

```diff
@@ -41,11 +51,18 @@ let package = Package(
                 .target(name: "Sham"),
             ]
         ),
+        .target(
+            name: "UtilityBeltUITestingHelpers",
```

Comment:
Same as the previous comment--I don't know what this module is for, by itself, so I wouldn't know when I need to import this.